lib/ansible/modules/windows/win_power_plan.ps1

```diff
@@ -0,0 +1,83 @@
+#!powershell
+# Copyright (c) 2017 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+#Requires -Module Ansible.ModuleUtils.Legacy
+
+$params = Parse-Args -arguments $args -supports_check_mode $true
+$check_mode = Get-AnsibleParam -obj $params -name "_ansible_check_mode" -type "bool" -default $false
+$diff_mode = Get-AnsibleParam -obj $params -name "_ansible_diff" -type "bool" -default $false
```

Comment:
Ok, I'll change it. I'm wondering though if I should leave this in and if so, do I need to do any kind of handling for it if my module doesn't do anything for diff mode?